manage-products.py

```diff
@@ -0,0 +1,99 @@
+"Separate GUI to manage the product info."
+
+import json
+from idlelib.textview import view_text
+from tkinter import *
+from tkinter import messagebox
+from tools.database import get_product_dict
+
+class ProductManager:
+    """
+    Main class for the interface. It doesn't
+    manage user information, so the product
+    info can be managed by anyone (if they know
+    how to do it!)...
+    """
+    
+    def __init__(self, root: Tk) -> None:
+        "constructor method."
+        self.root = root
+        self.product_dict = get_product_dict()
+        self.main_menu()
+    
+    def reload_product_info(self) -> None:
+        """
+        Open the file stream about product information. Even
+        when this stream does not affect the main form
+        ('Control de Agua.exe' and its Python format), it is
+        reccommended to close that program to use the new
+        values.
+        
+        (In most of the cases, you don't have to run this
+        method manually. This is only used to re-force the
+        product info).
+        """
+        del(self.product_dict)
+        self.product_dict = get_product_dict()
```

Comment:
I created this to force the GUI to reload the product info, to ensure a good syncing. I think `water-gui.py` can also implement this.